src/mds/MDSRank.cc

```diff
@@ -1210,6 +1212,11 @@ bool MDSRank::handle_deferrable_message(const cref_t<Message> &m)
       locker->dispatch(m);
       break;
 
+    case MSG_MDS_PING:
+      ALLOW_MESSAGES_FROM(CEPH_ENTITY_TYPE_MDS);
+      server->dispatch(m);
```

Comment:
Wouldn't we also want to fast dispatch these?

Reply:
I'm confused @vshankar, I'm looking at the current code and this is still done through regular dispatch? Did you forget to commit some changes?